sites/all/themes/pul_base/.stylelintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "stylelint-config-standard-scss",
+  "defaultSeverity": "warning",
+  "maxWarnings": 1453
```

Comment:
stylelint is much more picky than sass-lint was -- it has found 1,453 errors that it can't resolve automatically.  It's likely not a priority for us to resolve all of these, so lines 3-4 allow CI to still pass.